### DISCUSSION TITLE
登陆账号提示panic: runtime error: makeslice: len out of range

### DISCUSSION THREAD

**Discussion by ccjk45415** [Mrs4s/go-cqhttp#1515] [Q&A]
![DM)`YCFF_HZNF4NUF_M8 HG](https://user-images.githubusercontent.com/76248184/170623731-8009a56b-4777-41e2-9074-198fe525cd97.png)
求助 显示这个 用的v1.0.0-rc1版本的go-cqhttp

**Comment by Divename** [comment]
清理一下硬盘试试，我清理了硬盘就没事了
